templates/includes/installbox.hbs

```diff
@@ -1,7 +1,10 @@
 {{#with this}}
-  <a href="{{#if reference}}{{reference}}{{else}}{{name}}{{/if}}"><div class="install-box">
+<a href="{{#if reference}}{{reference}}{{else}}{{name}}{{/if}}">
+  <div class="install-box">
     <img src="../public/img/{{name}}.png" class="install-watch-img"><br>
     <b>{{models}}</b><br>({{name}}{{#if othernames}}{{#othernames}}, {{.}}{{/othernames}}{{/if}})<br>
-    <i>Support: {{#starString stars 5}}{{/starString}}</i>
-  </div></a>
-{{/with}}
+    <i>Support: {{#starString stars 5}}{{/starString}}</i><br>
+    {{#if releasedate}} <i>Release date: <b>{{releasedate}}</b></i>{{/if}}
```

Comment:
I fear this would take quite a bit of space in the install box, which we've been trying to keep concise. How about a small hint in parenthesis after the name ? It should be obvious that the year is the release date.

@eLtMosen any opinions on this ?